.github/workflows/job_tensorflow_layer_tests.yml

```diff
@@ -98,27 +100,13 @@ jobs:
           should-setup-pip-paths: ${{ runner.os == 'Linux' }}
           self-hosted-runner: ${{ runner.os == 'Linux' }}
 
-      - name: Install OpenVINO Python wheels (Linux and macOS)
-        if: runner.os != 'Windows'
-        run: |
-          # Install the core OV wheel
-          python3 -m pip install ./openvino-*.whl
-
-          # Install the core OV Tokenizers wheel
-          python3 -m pip install ./openvino_tokenizers-*.whl
-        working-directory: ${{ env.INSTALL_WHEELS_DIR }}
-
-      - name: Install OpenVINO Python wheels (Windows)
-        if: runner.os == 'Windows'
-        run: |
-          # Find and install the core OV wheel
-          $ovCoreWheelPath=Get-ChildItem -Path . -Filter openvino-*.whl | % { $_.FullName }
-          python3 -m pip install "$ovCoreWheelPath"
-
-          # Find and install the core OV Tokenizers wheel
-          $ovCoreWheelPath=Get-ChildItem -Path . -Filter openvino_tokenizers-*.whl | % { $_.FullName }
-          python3 -m pip install "$ovCoreWheelPath"
-        working-directory: ${{ env.INSTALL_WHEELS_DIR }}
+      - name: Install OpenVINO Python wheels
```

Comment:
There is a problem with this approach: since the idea is to use `--no-index` so that pip does not try to install any local wheels from PyPI, it cannot install the dependencies because it is forbidden from trying to access PyPI like in https://github.com/openvinotoolkit/openvino/actions/runs/11794716928/job/32855012693#step:7:43. The wheels have many dependencies that are hosted on the package index hence we cannot restrict the pip from accessing the index.

Reply:
Actually, to make sure that a local wheel is installed, internally we pass the expected package version like this `pip install openvino-dev==2025.0.0.dev20241113 --find-links...`

Reply:
I think that we should proceed with the current approach: using the `openvino openvino_tokenizers ...` as the input and iterating over the given wheel names for installation. 
Additionally, the `openvino_tokenizers` wheel has a different version for some reason: 
![image](https://github.com/user-attachments/assets/272c2455-99b9-4d29-9217-d893a36b6dc8)
 so we would not be able to use it for local installation. Also, it would be rather cumbersome to pass the version from the `build` job to other jobs in which the wheels are used.